heron/common/src/cpp/network/connection.cpp

```diff
@@ -44,7 +44,6 @@ Connection::Connection(ConnectionEndPoint* endpoint, ConnectionOptions* options,
   mOnNewPacket = NULL;
   mOnConnectionBufferEmpty = NULL;
   mOnConnectionBufferFull = NULL;
-  mOnConnectionBufferChange = NULL;
```

Comment:
Because we don't need it any more, as I already describe in this PR, we move the code out of hot path